lib/controllers/AssetController.ts

```diff
@@ -138,10 +138,10 @@ export class AssetController extends CRUDController {
     const assetId = request.getId();
     const refresh = request.getRefresh();
     const strict = request.getBoolean('strict');
-    const devices = (await this.assetService.getAsset(engineId, assetId))._source.deviceIds;
-    if (Array.isArray(devices)) {
-      for (const device of devices) {
-        await this.deviceService.unlinkAsset(device, { refresh, strict });
+    const devicesLinks = (await this.assetService.getAsset(engineId, assetId))._source.deviceLinks;
```

Comment:
Nitpicking: we try to avoid wrapping `await` calls in parenthesis.

Also IMHO it's more easy to always manipulate the same objects in the code base (for instance the entire asset) than subpart of our models

Reply:
Done (but I think it's more easy to manipulate a subpart when we only need the subpart instead of duplicate "._source.deviceLinks" in all place I need to access it.